Game/Scripts/ParticleBillboardAsistance.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "Scripting\Script.h"
+#include "RuntimeInclude.h"
+
+RUNTIME_MODIFIABLE_INCLUDE;
+
+class ComponentTransform;
+
+class ParticleBillboardAsistance : public Script
```

Comment:
```suggestion
class ParticleBillboardAssistance : public Script
```
don't forget to modify the file name